### ISSUE TITLE
default options always used

### ISSUE THREAD

**Issue by 2pha** [ahmadnassri/node-metalsmith-imagemin#60]
shouldn't:
```
let options = Object.assign({}, opts)
options = Object.assign(options, defaults)
```
be:
```
let options = Object.assign({}, opts)
options = Object.assign(defaults, options)
```
?

Also, as these defaults are always added to the active plugins, how would one go about not using one of the defaults? For example if I didn't want to use 'jpegrecompress' ?

**Comment by 2pha** [comment]
Hmm, seems I can't get this module to work at all anyway.
Maybe there is something I am missing? 
var imagemin = require('metalsmith-imagemin');
.use(imagemin())

**Comment by firyx** [comment]
I have created a pull request to fix this:
https://github.com/ahmadnassri/metalsmith-imagemin/pull/62

There will be an option to disable all defaults or disable/replace a specific plugin.
Usage is documented in README.md

**Comment by ahmadnassri** [comment]
sorry for the delay in addressing and thank you @firyx for the fix! it is now merged and released in [`v4.1.0`](https://github.com/ahmadnassri/metalsmith-imagemin/releases/tag/v4.1.0)
